src/gen_code/mod.rs

```diff
@@ -1,4 +1,9 @@
+//
+// Copyright 2020, Offchain Labs, Inc. All rights reserved.
```

Comment:
This should probably list the year as "2020-2021" because it appears to contain code written in both years.

Reply:
Good idea, I'm updating the copyright notice